rclpy/test/test_rosout_subscription.py

```diff
@@ -114,13 +114,6 @@ def call_logger(logger):
         self.executor.spin_until_future_complete(self.fut, 3)
         self.assertTrue(self.fut.done())
 
-    def test_node_logger_not_exist(self):
```

Comment:
I think we want to add a test to ensure that the situation from https://github.com/ros2/rclpy/issues/1195 doesn't occur again.  The following code (which needs to be adapted into a test) reproduced for me:

```
import rclpy
from rclpy.action import ActionServer

from example_interfaces.action import Fibonacci

class MyClass:
    def __init__(self, node):
        self.action_server = ActionServer(
            node,
            Fibonacci,
            "/fibonacci_action_1",
            self.execute_callback,
        )

    def execute_callback(self):
        print("Execute callback")

rclpy.init()
node = rclpy.create_node("mynode", enable_rosout=False)
x = MyClass(node)
node.destroy_node()
rclpy.try_shutdown()
```

Reply:
Thanks. 
I used the similar logic to test the case. Updated in https://github.com/ros2/rclpy/pull/1196/commits/fc84c5e18c1655d2a0ed88dc4a04ef3d295d8459.